scripts/e2e-docker-start

```diff
@@ -21,12 +21,18 @@ fi
 
 echo "Using Rancher image version: ${RANCHER_IMG_VERSION}"
 
+# Configure the Rancher server URL from start, similarly to how chart installations work
+# The special host.docker.internal domain only works for Docker Desktop, not plain Docker Engine on linux, so using the special IP 172.17.0.1 to access the gateway of the default bridge network
+SERVER_URL_ARGS="-e CATTLE_SERVER_URL=https://172.17.0.1"
+#
 # we are now enabling the "oidc-provider" feature flag on install because we will need it for e2e testing (not enabled by default in non-prime versions)
 # "oidc-provider" feature flag needs a system restart so it's not feasible to enable it as part of the test
-docker run -d --restart=unless-stopped -p 80:80 -p 443:443 ${VOLUME_ARGS} \
+
+docker run -d --restart=unless-stopped -p 80:80 -p 443:443 -p 6060:6060 ${VOLUME_ARGS} \
```

Comment:
can we pull out `-p 80:80 -p 443:443 -p 6060:6060` as `PORT_ARGS` and add a comment on why 6060 is needed?